install/kubernetes/helm/istio-remote/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: istio-multi
+  labels:
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: istio-reader
+subjects:
+- kind: ServiceAccount
+  name: istio-control-plane-service-account
```

Comment:
This service account isn't created in any chart that I can see.  Did you add a new file to create this service account but it didn't get added to changeset?